packages/graphql-language-service/src/client.ts

```diff
@@ -102,7 +102,7 @@ function _getAutocompleteSuggestions(
     process.stdout.write(JSON.stringify(resultObject, null, 2));
     return GRAPHQL_SUCCESS_CODE;
   } catch (error) {
-    process.stderr.write(error);
+    process.stderr.write(error.stack + '\n');
```

Comment:
Since we're already getting errors here, should we be more cautious? Something like:

```suggestion
    process.stderr.write((error?.stack ?? String(error)) + '\n');
```

This helps protect against someone throwing `undefined` (or, less likely, `null`):

```
> try { throw undefined; } catch (error) { process.stderr.write(error.stack + '\n'); }
Thrown:
TypeError: Cannot read property 'stack' of undefined
```

Reply:
@yoshiakis what do you think?

Reply:
Sorry for late. Yes @benjie, I like your idea!

How about these? (Please tell me if the error expression looks weird 'cause I'm not proud of my writing skill in English.)
```TypeScript
if (error) {
  process.stderr.write(error.stack + '\n');
} else {
  process.stderr.write('falsy value such as undefined was thrown' + '\n');
}
```

Reply:
looks good to me!

Reply:
actually now that i think about it, i think benjie's suggestion is most suitable. we want to use String(error) to allow different error types to stringify themselves, which includes printing the call stack

Reply:
Unfortunately, String(error) may not include the call stack. You can check this by putting `process.stderr.out(String(error) + '\n');` below `process.stderr.write(error.stack + '\n');`.

Reply:
`error?.stack ?? String(error)` should handle this: if error is set, and it has a stack property which is non-null and non-undefined, use this (i.e. the stack); otherwise fall back to `String(error)`

Reply:
@benjie You are right! I misunderstood what nullish coalescing operator (??) does.

I agree that benjie's suggestion is better than mine.